src/style.css

```diff
@@ -0,0 +1,74 @@
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.label {
+  display: block;
+  position: relative;
+  margin-top: 20px;
+}
+
+.input_placeholder {
+  padding-left: 62px;
+  width: 100%;
+  height: 70px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: Avenir-Book, serif;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 20px;
+}
+
+.input_placeholder_small {
+  height: 42px;
+  padding-left: 33px;
+  font-size: 14px;
+  line-height: 19px;
+}
+
+.img_search {
+  position: absolute;
+  top: 25px;
+  left: 26px;
+  height: 19px;
+  width: 19px;
+}
+
+.img_search_small {
+  position: absolute;
+  top: 15px;
+  left: 13px;
+  height: 11px;
+  width: 11px;
+}
+
+.input_placeholder:hover {
+  color: #3d4e61;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.input_placeholder:focus {
+  color: #3d4e61;
```

Comment:
If you use a color more than one time, move its value to the CSS constant and use it everywhere you need

Reply:
done